x-pack/test/functional/apps/ml/settings/filter_list_creation.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { asyncForEach } from './common';
+
+export default function ({ getService }: FtrProviderContext) {
+  const ml = getService('ml');
+  const testDataList = [1, 2, 3].map((n) => ({
+    filterId: `test_create_filter_${n}`,
+    description: `test description ${n}`,
+    keywords: ['filter word 1', 'filter word 2', 'filter word 3'],
+  }));
+
+  describe('filter list creation', function () {
+    before(async () => {
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+      // clean up created filters
+      await asyncForEach(testDataList, async ({ filterId }) => {
+        await ml.api.deleteFilter(filterId);
+      });
+    });
+
+    it('filter list creation creates new filter list', async () => {
+      await ml.testExecution.logTestStep(
+        'filter list creation loads the filter list management page'
+      );
+      await ml.navigation.navigateToMl();
+      await ml.navigation.navigateToSettings();
+      await ml.settings.navigateToFilterListsManagement();
+
+      await ml.testExecution.logTestStep('filter list creates new filter lists');
+      for (const testData of testDataList) {
```

Comment:
I don't see how going through the creation process three times in the same way adds value to the test. We usually only iterate through test data when we have different configurations that we want to test (e.g. with/without events, with/without description, ...), which would also require assertions for the specific scenarios. I'd suggest to just go with one filter list here.

Reply:
Updated here 20c6b2d